packages/ui/src/common/hooks/useSignAndSendTransaction.ts

```diff
@@ -50,5 +54,6 @@ export const useSignAndSendTransaction = ({ transaction, signer, service, skipQu
     paymentInfo,
     sign,
     isReady,
+    noFeeFunds: Boolean(balance && paymentInfo && getFeeSpendableBalance(balance).lt(paymentInfo.partialFee)),
```

Comment:
The logic is really nice! Again a PR which fix a bug while simplifying the code :raised_hands: 

I'm just confused by the `noFeeFunds` name. `hasFunds` was lot clearer IMO. Or maybe we could even use `canAfford` to be consistent with whats returned by `useTransactionFee` elsewhere.

WDYT ?

Reply:
Thanks! Sure, sounds better and seems more straightforward 😃